app/resonant-laboratory/grunt-watch-plugin.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
   apply: function (compiler) {
     compiler.plugin('emit', (compilation, callback) => {
       if (compilation.options.watch === true) {
-        exec('cd $GIRDER_PATH && grunt', function (error, stdout, stderr) {
+        exec('cd $GIRDER_PATH && grunt --force', function (error, stdout, stderr) {
```

Comment:
In that case I'd prefer we not just put `--force` in here. Let's try to figure out what's going wrong on your machine first.